snap/snapcraft.yaml

```diff
@@ -63,9 +64,13 @@ apps:
       - network-bind
       - removable-media
       - joystick
+      - mount-observe
+      - system-observe
+      - network-observe
+      - hardware-observe
     environment:
       SDL_VIDEODRIVER: wayland
-      LD_LIBRARY_PATH: "$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/:$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/libunity/:$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/pulseaudio/:$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/speech-dispatcher/"
+      LD_LIBRARY_PATH: "$LD_LIBRARY_PATH:$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/libunity/:$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/pulseaudio/:$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/speech-dispatcher/"
       ESPEAK_DATA_PATH: "$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/espeak-ng-data"
```

Comment:
Is it possible to remove `$LD_LIBRARY_PATH` here? According to 0c9d44faac68b1da45d33baf1b8901e7e079929b and the comments in the forums, this could potentially break stuff.

Reply:
Sure, I was just syncing it with the non-daemon app.